lib/cn/csched_sp3.c

```diff
@@ -1570,8 +1586,7 @@ sp3_submit(struct sp3 *sp, struct cn_compaction_work *w, uint qnum, uint rbt_idx
             HSE_SLOG_END);
     }
 
-    sp->activity++;
-
```

Comment:
I removed this because it seems to cause the monitor to loop unnecessarily...

Reply:
I think it was supposed to eliminate delay between starting jobs when a bunch of work came up at once.